mailbox/store/src/main/java/org/apache/james/mailbox/store/StoreMailboxManager.java

```diff
@@ -126,6 +127,9 @@
 
     private final MessageParser messageParser;
 
+    private int limitOfAnnotations;
```

Comment:
Make it final please, in both case I can now it at instanciation, and it will not change. (Optional constructor parameter might be the solution ?)